### PR TITLE
Update russian translations

### PR DESCRIPTION
**Pull request by oddbyte**
The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.

### CODE REVIEW

File: ru/translation.json [hunk 1]
```diff
@@ -260,8 +260,8 @@
 	},
 
 	"cookie_notice": {
-		"header": "Ooo! Cookies!",
-		"subheader": "We use cookies to allow you access to our site and services.<br>Cookies are bound by our Cookies are bound by our <a href='//%SITE_URL%/privacy'>%PRIVACYPOLICY%</a>.%CURSOR%",
-		"button": "OK, cool"
+		"header": "Ooo! Печенье!",
+		"subheader": "Мы используем файлы cookie чтобы предоставить вам доступ к нашему сайту и услугам.<br>Файлы cookie регулируются нашей <a href='//%SITE_URL%/privacy'>%ПОЛИТИКОЙ КОНФИДЕНЦИАЛЬНОСТИ%</a>.%CURSOR%",
+		"button": Хорошо, круто"
```
Comment (Aisuruneko) [inline ru/translation.json]: you left a " out here.

**Review by Aisuruneko — CHANGES_REQUESTED** [review]
You left a " out.

**Review by oddbyte — COMMENTED** [review]
The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.

File: ru/translation.json (hunk 1 above)
Comment (oddbyte) [reply ru/translation.json]: oh crap

Comment (oddbyte) [reply ru/translation.json]: will fix

Comment (oddbyte) [reply ru/translation.json]: fixed

The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.

**Review by Aisuruneko — APPROVED** [review]

**Comment by oddbyte** [comment]
The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.

**Comment by oddbyte** [comment]
ok now merge

**Pull request merged**